nodejs/week3/warmUp/index.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const app = express();
+const port = 3000;
+
+app.use(bodyParser.json());
+
+// Calculator using params
+app.get('/calculator/:param', (req, res) => {
+	const { param } = req.params;
+	const { firstParam } = req.query;
```

Comment:
you can say
	const { firstParam, secondParam } = req.query;

Reply:
Yes, I should have done. I was not that sure about de-structuring objects at the time of nodejs, but I understand it better now.